README.rst

```diff
@@ -1,22 +1,114 @@
-FIWARE Policy Manager GE
-____________________
+================================
+FIWARE Policy Manager GE - Cloto
+================================
 
 
 | |Build Status| |Coverage Status| |Pypi Version| |Pypi License|
 
 
-Description
-===========
+- `Introduction`_
 
-This module is part of FIWARE Policy Manager. It provides an API-REST to create rules associated to servers,
-subscribe servers to Context Broker to get information about resources consumption of that servers and launch actions
-described in rules when conditions are given.
+- `GEi overall description`_
+
+  * `Components`_
+
+- `Build and Install`_
+
+  * `Requirements`_
+  * `Installation`_
+  * `Configuration file - Cloto`_
+  * `Configuration - Apache + wsgi`_
+
+- `Running`_
+
+- `API Overview`_
+
+  * `API Reference Documentation`_
+
+- `Testing`_
+
+  * `Unit tests`_
+  * `End-to-end tests`_
+  * `Acceptance tests`_
+
+- `Advanced topics`_
+
+- `Support`_
+
+- `License`_
+
+
+Introduction
+============
+
+This is the code repository for **FIWARE Policy Manager GE - Cloto**, a server that provides an API-REST to
+create rules associated to servers, subscribe servers to Context Broker to get information about resources
+consumption of that servers and launch actions described in rules when conditions are given.
+
+Check also the `FIWARE Catalogue entry for Policy Manager <http://catalogue.fiware.org/enablers/policy-manager-bosun>`__
```

Comment:
This is a explicit link, so replace ``__`` (two underscores) by a single ``_``. And, given that it links to a URL instead to a local file, I'd prefer not using "embedded" link style (i.e., we'd better just include here ``label_``, and put the actual URL at the end of the document)